CRM/Core/Error.php

```diff
@@ -615,10 +618,7 @@ public static function debug_log_message($message, $out = FALSE, $prefix = '', $
     }
 
     if (!empty(\Civi::$statics[__CLASS__]['userFrameworkLogging'])) {
-      // should call $config->userSystem->logger($message) here - but I got a situation where userSystem was not an object - not sure why
-      if ($config->userSystem->is_drupal and function_exists('watchdog')) {
-        watchdog('civicrm', '%message', ['%message' => $message], WATCHDOG_DEBUG);
```

Comment:
For getting this merged upstream, we're probably going to have to leave this code here and maybe put your code in an `else`?